# HG changeset patch
# User Andrew John Hughes <ahughes@redhat.com>
# Date 1338297380 -3600
# Node ID fb4e64ae06a31c878cc711579fe87a6f626f73fb
# Parent  2c3c929e038c6c64d79ecf98005f5fd2f4d0f37d
Don't add a suffix for building the default, IcedTea.

2012-05-25  Andrew John Hughes  <ahughes@redhat.com>

	* Makefile.am:
	(versioning): Don't add a suffix if building
	the default project, 'icedtea'.

diff --git a/Makefile.am b/Makefile.am
--- a/Makefile.am
+++ b/Makefile.am
@@ -1435,7 +1435,7 @@
 	  echo "DISTRO_PACKAGE_VERSION=$(PKGVERSION)" \
 	    >>openjdk/jdk/make/common/shared/Defs.gmk ;
 endif
-	if test x"$(PROJECT_NAME)" != "xjdk7"; then \
+	if test x"$(PROJECT_NAME)" != "xjdk7" && test x"$(PROJECT_NAME)" != "xicedtea"; then \
 	  proj_suffix="-$(PROJECT_NAME)"; \
 	fi ; \
 	if test x"$(VERSION_SUFFIX)" != "x"; then \
